src/site/twitch.tv/modules/player/PlayerController.vue

```diff
@@ -68,12 +68,31 @@ watchEffect(() => {
 		);
 		if (!playerOverlay.value) return;
 
-		if (actionOnClick.value === 1) {
-			defineNamedEventHandler(playerOverlay.value, "PlayerActionOnClick", "click", togglePause);
-		} else if (actionOnClick.value === 2) {
-			defineNamedEventHandler(playerOverlay.value, "PlayerActionOnClick", "click", toggleMute);
+		// using xpath to get the `LIVE` indicator on the player to determine if channel is live or not
+		const isLiveOrVOD = document.evaluate(
+			"//div[contains(@class, 'persistent-player')]//div[contains(@class, 'tw-channel-status-text-indicator')]/p",
```

Comment:
It's unclear why use an extremely old web technology when better alternatives, such as `document.querySelector()`, exist.